<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connexion</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="placeholderText">
      <string>Nom d'utilisateur</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Mot de passe</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_connexion">
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_motdepasseoublie">
     <property name="text">
      <string>Mot de passe oubli√© ?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
